pants

```diff
@@ -83,18 +84,19 @@ PANTS_SRCPATH=(
   "${HERE}/src/python"
 )
 if [[ -n "${WRAPPER_SRCPATH}" ]]; then
+  # shellcheck disable=SC2207
   PANTS_SRCPATH=(
     $(echo "${WRAPPER_SRCPATH}" | tr : ' ')
     "${PANTS_SRCPATH[@]}"
   )
 fi
-PANTS_SRCPATH="$(echo "${PANTS_SRCPATH[@]}" | tr ' ' :)"
+PANTS_SRCPATH_STR="$(echo "${PANTS_SRCPATH[@]}" | tr ' ' :)"
```

Comment:
It's a bad practice to reassign an array to a string, according to shellcheck, so we simply workaround this via the new variable name.

Reply:
I would think it's probably "bad practice" to reassign a variable with a different type regardless of the type? So this seems reasonable.

Reply:
I'm confused, here we were assigning a String to an array, right? Just checking my understanding.

Reply:
Originally, we were converting the array `PANTS_SRCPATH` into a string. Now, we are creating a new string variable from the array `PANTS_SRCPATH`.